src/section/bootcamp/view/view.tsx

```diff
@@ -0,0 +1,250 @@
+import { FC, useEffect } from 'react'
+import { useParams, Link } from 'react-router-dom'
+import { ICON_NAME, useIcon } from 'hook'
+import { useGetBootcampQuery, useGetAllCourseByBootcampQuery } from 'store/slice'
+import {
+  Box,
+  Grid,
+  Typography,
+  Container,
+  CardMedia,
+  Chip,
+  CardContent,
+  Divider,
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  List,
+  ListItem,
+  ListItemText
+} from '@mui/material'
+import { BackButton } from 'component'
+import { GSContainerGrid, SSpanBox } from 'theme/style'
+import { MotionLazyContainer } from 'component/motion'
+import { ICON_WEB_NAME } from 'config'
+import { FLEX, KEY, LABEL, PLACEHOLDER, TYPOGRAPHY, COMPONENT, ARIA, VARIANT, SIZE } from 'constant'
+import { badgeLocation, photoLocation } from 'util/asset-loc'
+import CourseTable from './course-table'
+
+interface BootcampViewSectionProps {
+  bootcamp?: Bootcamp
+}
+
+const BootcampViewSection: FC<BootcampViewSectionProps> = () => {
+  const { id } = useParams<{ id: string }>()
+  const { data: bootcamp, error, isLoading } = useGetBootcampQuery(id)
+
+  const {
+    _id,
+    name,
+    description,
+    badge,
+    photo,
+    location,
+    careers,
+    averageRating,
+    user,
+    duration,
+    phone,
+    website,
+    slug,
+    housing,
+    jobAssistance,
+    jobGuarantee
+  } = bootcamp?.data || {}
+
+  const courseQueryResult = useGetAllCourseByBootcampQuery(_id)
+
+  const { data: course, error: courseError, isLoading: courseLoading } = courseQueryResult || { data: null, error: null, isLoading: false }
+
+  const { Icon: PhoneIcon, iconSrc: phoneSrc } = useIcon(ICON_WEB_NAME.PHONE)
+  const { Icon: RightIcon, iconSrc: rightSrc } = useIcon(ICON_WEB_NAME.CHEVRON_RIGHT)
+  const { Icon: GlobeIcon, iconSrc: globeSrc } = useIcon(ICON_WEB_NAME.GLOBE)
+  const { Icon, iconSrc: crossSrc } = useIcon(ICON_WEB_NAME.CLOSE_OUTLINE)
+  const { iconSrc: checkSrc } = useIcon(ICON_WEB_NAME.CHECK_OUTLINE)
+
+  return (
+    <MotionLazyContainer>
+      <Box
+        sx={{
+          height: '100vh'
+        }}>
+        <Grid container spacing={2}>
+          <Grid item sm={12}>
+            <BackButton />
+          </Grid>
+        </Grid>
+        <Container>
+          <Grid container spacing={2} flexDirection='row'>
+            <Grid item sm={9}>
+              <Box marginBottom={2}>
+                <Typography variant={TYPOGRAPHY.H4} color='common.black'>
+                  {name}
```

Comment:
clean/refactor this up, will create a ticket for this.

Reply:
noted. #163